src/main/java/mate/academy/service/order/OrderServiceImpl.java

```diff
@@ -0,0 +1,50 @@
+package mate.academy.service.order;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.dto.order.OrderDto;
+import mate.academy.dto.order.OrderRequestDto;
+import mate.academy.exception.EntityNotFoundException;
+import mate.academy.mapper.OrderMapper;
+import mate.academy.model.Order;
+import mate.academy.repository.OrderRepository;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class OrderServiceImpl implements OrderService {
+    private final OrderRepository orderRepository;
+    private final OrderMapper orderMapper;
+
+    @Override
+    public OrderDto createOrder(OrderRequestDto requestDto) {
+        Order order = orderMapper.toModel(requestDto);
+        order.setOrderDate(LocalDateTime.now());
+        return orderMapper.toDto(orderRepository.save(order));
```

Comment:
Is it enough? You should calculate the total price and clear ShoppingCart. Don’t  forget about Transactional 
Did you test your solution?